### PR TITLE
TV

### PR DESCRIPTION
**Pull request by wtvamp**
Check out Emanuel's class and tell us what it looks like

### CODE REVIEW

File: src/gamesupport/TV.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+public class TV {
+    public int size;
+    private int power;
+    public int channel;
+    private int maxChannel;
+
+    public TV(int inches, int wattage) {
+        size = inches;
+        power = wattage;
+        // my favorite number is 8 so every TV will start on this channel
```
Comment (jmorales8) [inline src/gamesupport/TV.java:14]: Nice number!

**Review by jmorales8 — APPROVED** [review]

File: src/gamesupport/TV.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+            channel = channel - 1;
+        }
+        System.out.println("the TV changed the channel to " + channel);
+    }
+
+    public void changeChannel(int channelNumber) {
+        channel = channelNumber;
+        System.out.println("the TV changed the channel to " + channel);
+    }
+
```
Comment (GrpFruit) [inline src/gamesupport/TV.java:44]: nice I like this

**Review by GrpFruit — APPROVED** [review]

**Comment by ajcruzmart** [comment]
Nice code!

**Review by MegaBrute — APPROVED** [review]

File: src/gamesupport/TV.java [hunk 3]
```diff
@@ -0,0 +1,53 @@
+package gamesupport;
+
+import java.text.MessageFormat;
+
+public class TV {
+    public int size;
+    private int power;
```
Comment (ajcruzmart) [inline src/gamesupport/TV.java:7]: NIce Code!

**Review by BreadToastt — APPROVED** [review]
got y'all feedback thx

**Pull request merged**